src/tsolvers/lasolver/Simplex.cc

```diff
@@ -316,13 +316,16 @@ void Simplex::updateValues(const LVRef bv, const LVRef nv){
 Simplex::Explanation Simplex::getConflictingBounds(LVRef x, bool conflictOnLower)
 {
     LABoundRef br_f = conflictOnLower ? model->readLBoundRef(x) : model->readUBoundRef(x);
-    Explanation expl = {{br_f,1}};
+    auto const & row = tableau.getRowPoly(x);
+    Explanation expl;
+    expl.reserve(row.size() + 1);
+    expl.push_back({br_f,1});
```

Comment:
Why not emplace_back?

Reply:
Because my compiler refused to compile that :)
I guess we would have to add a constructor to `ExplTerm` and I did not feel like doing that.
I am hoping compilers will do their magic and optimize this anyway...